ui/src/components/sermons/CurrentSermon.js

```diff
@@ -76,14 +121,18 @@ const CurrentSermon = ({ currentSermon }) => {
             ratio={16 / 9}
             display={{ base: "none", md: "usnet" }}
           >
-            <iframe
-              width="560"
-              height="315"
-              src={`https://www.youtube.com/embed/${videoUrl}`}
-              title="YouTube video player"
-              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-              allowFullScreen
-            ></iframe>
+            {isOnline ? (
```

Comment:
if you have time and wanna do it, i think this file (and the other sermon files) need a major refactor. There are lots of repeated code